ML_PCA_from_Scratch/pca.py

```diff
@@ -0,0 +1,61 @@
+from numpy import mean,std,cov
```

Comment:
Fix folder name.

Reply:
I am unsure about the naming convention. Please help me out